### PR TITLE
chore: added quote to community/community-managers page

### LINKED ISSUE #4579: Add a quote to the Layer5.io/community/community-managers page
### Desired Behavior
We want to include a quote in the [layer5.io](https://layer5.io)  `/community/community managers` page.

> "Thank you so much for your kind words. I absolutely love this community"

See similar merged PR for reference #4562 

### Screenshots / Mockups
[See Mockup](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI?type=design&node-id=16580-40239&mode=design&t=8zljk0lV92gX8jUQ-4)

---
<h2>Contributor Resources and <a href="https://layer5.io/community/handbook">Handbook</a></h2>

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- 📚 See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md).
- 🎨 Wireframes and designs for Layer5 site in [Figma](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs).
- 🙋🏾🙋🏼 Questions: [Discussion Forum](https://discuss.layer5.io) and [Community Slack](https://slack.layer5.io).

Join the Layer5 Community by submitting your [community member form](https://layer5.io/newcomer).

### PR DESCRIPTION
**Pull request by vrinda-mahajan**
**Description**

This PR fixes #4579 

**Notes for Reviewers**
I have added the inline quote to [https://layer5.io/community/community-managers](https://layer5.io/community/community-managers) after the manager's profile card grid.
Screenshot (263)

In the `src/components/Inline-quotes/index.js` file, I have made the following changes
- Added a new prop called `onlyQuoteIsPresent` to the `QuotesWrapper` styled component. This prop is used to conditionally style the h4 element that contains the quote. When onlyQuoteIsPresent is true, that is, none of the `image`, `person`, and `title`  props are present, the h4 element will occupy 100% width and be aligned to the center instead of the default 65% width, and right alignment.
- and also checks if only the `quote` prop is present then it will not render the `hr` tag.

Let me know if any changes are required.
**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit fe7f28584aace5d96f81316fb892cf71e26d336d at: https://64b82a2e54c1320e754e01ae--layer5.netlify.app

File: src/components/Inline-quotes/index.js [hunk 1]
```diff
@@ -24,8 +24,8 @@ text-align: center;
     }
 
     h4 {
-        text-align: right;
-        flex: 0 0 65%;
+        text-align: ${props => props.onlyQuoteIsPresent ? "center" : "right"};
```
Comment (leecalcote) [inline src/components/Inline-quotes/index.js:27]: Have you confirmed that this doesn't negatively impact other quotes?

**Review by leecalcote — COMMENTED** [review]
Thank you, @vrinda-mahajan

**Review by leecalcote — COMMENTED** [review]
Thank you, @vrinda-mahajan

File: src/components/Inline-quotes/index.js (hunk 1 above)
Comment (vrinda-mahajan) [reply src/components/Inline-quotes/index.js:27]: Yes, I have confirmed that all the other quotes are as they were before.

Comment (leecalcote) [reply src/components/Inline-quotes/index.js:27]: Wonderful. 🙂

**Pull request merged**